rebar.config

```diff
@@ -29,6 +29,8 @@
 
 {minimum_otp_vsn, "21.0"}.
 
+{escript_emu_args, "%%! -connect_all false \n" }.
```

Comment:
I don't have a problem with this, but how come this is an issue for you? Isn't a node "added" only at the first interaction (e.g. an explicit ping, or a remote spawn)? I imagine the language server node would not try to connect to other nodes. But maybe in your system the other nodes are trying to connect to it somehow?